PendingReleaseNotes

```diff
@@ -68,6 +68,15 @@
   is enabled. This helps with the monitor logs on larger clusters, that may get
   many 'osd.X reported immediately failed by osd.Y' messages, and confuse tools.
 
+* The mclock scheduler is now more refined with a set of built-in profiles that provides QoS
+  between the external and internal clients of Ceph. The mclock scheduler can be enabled by
+  setting the config option "osd_op_queue" to "mclock_scheduler". The "high_client_ops" profile
+  is enabled by default. This profile allocates a larger chunk of osd bandwidth to external
+  client ops when compared to ops generated by internal clients like backround recovery, scrubs
+  etc.. In addition, a couple of additional built-in profiles i.e. "high_recovery_ops" and
+  "balanced" may be enabled based on the requirement. The built-in profiles once enabled sets
+  the required Ceph options so as to optimize the QoS provided to clients of mclock scheduler.
+
```

Comment:
```suggestion
The mclock scheduler has been refined. A set of built-in profiles are now available that provide QoS between the internal and external clients of Ceph. To enable the mclock scheduler, set the config option "osd_op_queue" to "mclock_scheduler". The "high_client_ops" profile is enabled by default, and allocates more OSD bandwidth to external client operations than to internal client operations (such as background recovery and scrubs). Other built-in profiles include "high_recovery_ops" and "balanced". These built-in profiles optimize the QoS provided to clients of mclock scheduler.
```

Reply:
Thanks @zdover23. The suggestion LGTM. I will push this in my next update.